tests/test_mpt_gen.py

```diff
@@ -143,3 +141,52 @@ def test_mpt_generate_callback(attn_impl: str, use_alibi: bool,
 
     generate.generate.assert_called_once()
     trainer.logger.log_table.assert_called_once()
+
+
+@pytest.mark.gpu
+@pytest.mark.parametrize('attn_impl', ['triton', 'torch'])
+def test_mpt_generate_callback_not_tied(
+        attn_impl: str, build_tiny_mpt: Callable[..., ComposerMPTCausalLM],
+        tiny_ft_dataloader: DataLoader):
+    device = get_device('gpu')
+
+    # build mpt model
+    model = build_tiny_mpt(
+        tie_word_embeddings=False,
+        attn_config={
+            'attn_impl': attn_impl,
+            'attn_uses_sequence_id': False,
+            'alibi': True,
```

Comment:
nit: does this test fail when alibi is True and tied word embeddings False? If so, would prefer we keep it in the same function and just assert that an error is thrown in this case that we don't support atm.

Reply:
nah, there was some flakiness associated with this test so I split it into two in case some weird pytest stuff was happening, and alibi + untied word embeddings didn't seem like a relevant different thing to test so I removed it. I can add it back though.

Reply:
I'll add it back just to make sure